image/image.html

```diff
@@ -3,12 +3,54 @@
     category: 'output',
     color: '#a6bbcf',
     defaults: {
+      active: {value:true},
       name: { value: "" }
     },
     inputs: 1,
     outputs: 0,
     icon: "watch.png",
     align: 'right',
+    button: {
+        toggle: "active",
+        onclick: function() {
+            var label = this.name||"debug";
+            var node = this;
+            $.ajax({
+                url: "image-output/"+this.id+"/"+(this.active?"enable":"disable"),
```

Comment:
[Template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) would probably make this look a bit nicer :)

Reply:
I agree ...